core/src/test/scala/org/apache/spark/scheduler/SparkListenerSuite.scala

```diff
@@ -255,21 +262,19 @@ class SparkListenerSuite extends SparkFunSuite with LocalSparkContext with Match
         taskInfoMetrics.map(_._2.executorDeserializeTime),
         stageInfo + " executorDeserializeTime")
 
-      /* Test is disabled (SEE SPARK-2208)
       if (stageInfo.rddInfos.exists(_.name == d4.name)) {
         checkNonZeroAvg(
           taskInfoMetrics.map(_._2.shuffleReadMetrics.get.fetchWaitTime),
           stageInfo + " fetchWaitTime")
       }
-      */
 
       taskInfoMetrics.foreach { case (taskInfo, taskMetrics) =>
-        taskMetrics.resultSize should be > (0L)
+        taskMetrics.resultSize should be > 0L
```

Comment:
This:
taskMetrics.resultSize should be > 0L
 is analogous to:
taskMetrics.resultSize.should(be.>(0L))